cfme/fixtures/cli.py

```diff
@@ -1,17 +1,26 @@
-import pytest
-
 from collections import namedtuple
 from contextlib import contextmanager
+
+import fauxfactory
+import pytest
 from six import iteritems
 
 import cfme.utils.auth as authutil
```

Comment:
sort imports alphabetically please .
This can move to line 15

Reply:
I believe this is the correct ordering as its 'import' not 'from'